pipeline/batch/sql_tasks.py

```diff
@@ -29,7 +29,8 @@
     'QA': ('Location', 'http://www.vodafone.qa/alu.cfm'),
     'KR': ('Location', 'http://www.warning.or.kr'),
     'PT': ('Location', 'http://mobilegen.vodafone.pt/denied/dn'),
-    'NO': ('Location', 'http://block-no.altibox.net/')
+    'NO': ('Location', 'http://block-no.altibox.net/'),
+    'UK': ('Location', 'http://blocked.nb.sky.com%')
```

Comment:
If we could support a regular expression this expression `block.*\.nb\.sky\.com` will trigger almost double amount of block pages.